### PR TITLE
Fix to use airflow 2.0 logging configuration

### LINKED ISSUE #49: Support airflow 2.0
Has anyone used this helm chart with airflow 2.0 yet? 
I have been trying with airflow by replacing **1.10.12-python3.6** to **2.0.0-python3.6**
but get this error 

`airflow command error: argument GROUP_OR_COMMAND: `airflow upgradedb` command, has been removed, please use `airflow db upgrade`, see help above.`

So friendly asking the plan to support airflow 2.0?

### PR DESCRIPTION
**Pull request by Swalloow**
Signed-off-by: Swalloow <swalloow.me@gmail.com>

**What issues does your PR fix?**

- fixes #49 (DeprecationWarning messages)

**What does your PR do?**

```
/home/airflow/.local/lib/python3.6/site-packages/airflow/configuration.py:320 DeprecationWarning: The base_log_folder option in [core] has been moved to the base_log_folder option in [logging] - the old setting has been used, but please update your config.
/home/airflow/.local/lib/python3.6/site-packages/airflow/configuration.py:320 DeprecationWarning: The dag_processor_manager_log_location option in [core] has been moved to the dag_processor_manager_log_location option in [logging] - the old setting has been used, but please update your config.
/home/airflow/.local/lib/python3.6/site-packages/airflow/configuration.py:320 DeprecationWarning: The remote_logging option in [core] has been moved to the remote_logging option in [logging] - the old setting has been used, but please update your config.
/home/airflow/.local/lib/python3.6/site-packages/airflow/configuration.py:320 DeprecationWarning: The remote_base_log_folder option in [core] has been moved to the remote_base_log_folder option in [logging] - the old setting has been used, but please update your config.
```

Fix to use airflow 2.0 logging configuration.
Remove warning messages.
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#logging

## Checklist

- [x] Commits are [signed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#sign-your-work)
- [x] Commits are [squashed](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#squash-commits) (if appropriate)
- [x] Chart [version bumped](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#versioning)
- [x] Chart [documentation updated](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#documentation)
- [x] Passes [linting](https://github.com/airflow-helm/charts/tree/main/CONTRIBUTING.md#linting)

### CODE REVIEW

**Comment by chattarajoy** [comment]
@Swalloow you'll need to bump up the chart version, lint is failing because of that.

**Comment by gsemet** [comment]
Look ok for me, just yes bump the patch level in the chart version.

**Comment by Swalloow** [comment]
Thanks, I updated the chart version. But the following error occurred in the chart-testing task.
`sqlalchemy.exc.ProgrammingError: (psycopg2.errors.UndefinedTable) relation "connection" does not exist`

File: charts/airflow/UPGRADE.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+## `v7.16.0` → `v7.16.1`
+
+__The following IMPROVEMENTS have been made:__
+* We now use `AIRFLOW__LOGGING` instead of `AIRFLOW__CORE` configuration:
+  * `AIRFLOW__CORE__REMOTE_LOGGING`
+    * Is now `AIRFLOW__LOGGING__REMOTE_LOGGING`
+  * `AIRFLOW__CORE__REMOTE_BASE_LOG_FOLDER`
+    * Is now `AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER`
+  * `AIRFLOW__CORE__REMOTE_LOG_CONN_ID`
+    * Is now `AIRFLOW__CORE__REMOTE_LOG_CONN_ID`
```
Comment (mthoretton) [inline charts/airflow/UPGRADE.md:12]:
```suggestion
    * Is now `AIRFLOW__LOGGING__REMOTE_LOG_CONN_ID`
```

**Comment by thesuperzapper** [comment]
lets hold off on this, and do a full 2.0 fix release

**Comment by yoavo-datricks** [comment]
HI @thesuperzapper
I was wondering if you have any estimation as to when the helm will support airflow 2 as well.

Thanks!

**Comment by thesuperzapper** [comment]
@yoavo-datricks I am planning to take a serious crack at it in the next few days.

Given so many people use this chart in production, we need to make sure things like upgrading work properly.

Also, since as soon as we push out 2.0 support (by default), many people will adopt it in their environments, it's probably a good thing that we have waited for any initial issues with airflow 2.0 to emerge.

**Comment by joshi95** [comment]
Hi any progress on this ?

**Comment by thesuperzapper** [comment]
Still finishing it up (give me a day or 2) but changes from this PR have been incorporated in my local version, so I will close this PR.

**Comment by joshi95** [comment]
> ve me a day or

Thanks looking forward to it.

**Comment by joshi95** [comment]
> Still finishing it up (give me a day or 2) but changes from this PR have been incorporated in my local version, so I will close this PR.

Thanks looking forward to it.

**Pull request closed without merging**